src/App/App.js

```diff
@@ -3,37 +3,42 @@
 
 // SPDX-License-Identifier: MIT
 
+/* eslint-disable jsx-a11y/aria-role */
+
 import React, { Component } from 'react';
 import { BrowserRouter as Router, Route, Link } from 'react-router-dom';
 
-import Accounts from '../Accounts';
 import Send from '../Send';
+import Receive from '../Receive';
 import Tokens from '../Tokens';
 import './App.css';
 
 class App extends Component {
   render () {
     return (
       <Router>
-        <div className='App'>
-          <h1>Light Wallet</h1>
-          <ul>
-            <li>
-              <Link to='/'>Accounts</Link>
-            </li>
-            <li>
-              <Link to='/tokens'>Tokens</Link>
-            </li>
-            <li>
-              <Link to='/send'>Send</Link>
-            </li>
-          </ul>
-
-          <hr />
+        <div className='wrapper' role='layout'>
```

Comment:
I get a warning that this role="layout" is not [standard HTML5](http://knowledge.onsubject.com/2011/02/06/html-role-attributes/).

Since the [role="layout"] is only used in CSS, I think it would be more relevant here to use `<div className="wrapper layout">` and `.wrapper.layout` in the CSS